### PR TITLE
Alternative To Removing Kudzu Manufacturing (#30080)

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
:cl: Cobby
balance: Planting kudzu now has a short delay with a visible message to users around you. Hit-N-Run planting is no longer possible.
del: kudzu bluespace mutation and spacewalk mutation are removed.
/:cl:

closes #30080

This nerf strikes at 2 major problems, Hit-N-Run planting and kudzu going through walls/space, while still making it a station destroyer if done properly and with more effort than left clicking seeds while inhand.

If this is still unmanageable I'll lower the spread by 25% but I think this will be fine tbh.

Oh I did test this but my ~~github~~ local branch is out of date due to not using this computer in forever and so I had to transfer it from desktop to online via web editor. pls no bully gunhog

### CODE REVIEW

**Comment by Kromgar** [comment]
That thorny black kudzu should still break down doors shouldn't it?

**Comment by Tokiko1** [comment]
If I remember correctly there is a mutation that ex_act's stuff too, which means that it breaks doors/walls. Maybe.

**Comment by lordpidey** [comment]
I've seen kudzu break down walls somehow, it's probably the ex_act mutation @Tokiko1 mentioned, but let's double check.

**Comment by DoctoreMad** [comment]
Explosive kudzu mutations are horrible machines created by Moloch form his throne in hell and sent towards space to destroy the world of spessmen. I'm surprised that bluespace is being removed but that meme isn't.

Is it because none of the coders died to a ex-act kudzu vine explosion yet?

**Comment by CPTANT** [comment]
These changes are good and sufficient nerfs for now.

Lets not over-nerf something for a change.

**Comment by XDTM** [comment]
Aggressive spreading calls ex_act, along with esplosive kudzu. The latter should be removed mostly because there's no way to really fight it.

**Comment by Grazyn** [comment]
I've seen explosive kudzu level entire departments in a couple seconds with its chain reaction. Like literally leveled, everything in a huge area destroyed leaving only ruined plating behind. Imagine how an area looks after a massive blob is killed, that was it.

**Comment by jughu** [comment]
Keep the growing trough doors kudzu just add a delay to planting

**Comment by Kromgar** [comment]
Maybe instead of ex_act kudzu make it act like blob tiles that just attack things around them

**Comment by Cyberboss** [comment]
Remove the doors thing, don't nerf it into the ground

**Comment by ExcessiveUseOfCobblestone** [comment]
I removed spacewalking kudzu too because lol finding it in space. 

However oddly enough that airlock check was only in the space walk mutation yet applied to normal kudzu. how weird.

**Comment by ExcessiveUseOfCobblestone** [comment]
>Explosive kudzu mutations are horrible machines created by Moloch form his throne in hell and sent towards space to destroy the world of spessmen. I'm surprised that bluespace is being removed but that meme isn't.
>Is it because none of the coders died to a ex-act kudzu vine explosion yet?

Well this PR was to attack the problems with kudzu found in PR that's referenced in the title. Not a single post on said PR mentioned explosives, but several people suggested removing bluespace.

With that said, you can see how easy it is to remove a mutation, and this PR shows you the exact file to go to. If it bothers you that much, you're more than welcome to do your own PR removing that mutation as well.

**Comment by XDTM** [comment]
Fair enough, i usually ask for additions because that way there's no need to wait for another 24 hours to merge.

Either here or by someone else though, explosive kudzu should really get a different trigger or removed altogether, because of the lack of counters to it.

**Comment by ExcessiveUseOfCobblestone** [comment]
I personally would like to see how this goes before further nerfing it, hence my hesitation.

**Comment by Arianya** [comment]
Gonna mirror comment here that kudzu need to have their "able to live in space" feature nerfed or removed entirely. Anecdotally it causes MC issues, and less anecdotally it is a humongous pain in the ass to deal with since it expands out till the edge of the z-level with no real barrier.

**Comment by ExcessiveUseOfCobblestone** [comment]
@Arianya it removes the mutation entirely

**Comment by CPTANT** [comment]
Kudzu only growing in space adjacent to the station would be good.

File: code/modules/hydroponics/grown/kudzu.dm [hunk 1]
```diff
@@ -42,8 +42,10 @@
 	qdel(src)
 
 /obj/item/seeds/kudzu/attack_self(mob/user)
-	plant(user)
-	to_chat(user, "<span class='notice'>You plant the kudzu. You monster.</span>")
+	user.visible_message("<span class='danger'>[user] begins throwing seeds on the ground...</span>")
+	if(do_after(user, 50, needhand = 1, target = user, progress = 1))
```
Comment (Cyberboss) [inline code/modules/hydroponics/grown/kudzu.dm]: `needhand = TRUE, target = get_turf(user), progress = TRUE`

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**